tests/cases.py

```diff
@@ -657,7 +661,13 @@ def test_scores(self):
 
             # calculate manually
             scores_f_manual = self._exp_score(**kwargs).view(-1)
-            assert torch.allclose(scores_f_manual, scores_f), f"Diff: {scores_f_manual - scores_f}"
+            if not torch.allclose(scores_f, scores_f_manual, rtol=self.rtol, atol=self.atol):
+                # allclose checks: | input - other | < atol + rtol * |other|
+                a_delta = (scores_f_manual - scores_f).abs()
+                r_delta = (scores_f_manual - scores_f).abs() / scores_f.abs().clamp_min(1.0e-08)
```

Comment:
Should this clamp parameter be stored as a variable too?

Reply:
Right now, I do not see a reason why we should vary this between different models, so, for now, I would leave it as it is.